### PR TITLE
second commit

### PR DESCRIPTION
**Pull request by moisaoana**

### CODE REVIEW

File: main.c [hunk 1]
```diff
@@ -1,6 +1,128 @@
...
+#define MAX_FOOD_NAME 10
+#define MAX_TYPE_NAME 20
+void readType(char * model);
+double readPrice();
 
 int main() {
-    printf("Hello, World!\n");
+    int noOfFoods, noOfDrinks;
+    char ** foods, **drinks;
+    int *noOfTypes=(int*)malloc(noOfFoods*sizeof(int));
```
Comment (Ranapop) [inline main.c:11]: noOfTypes would be even better

File: main.c [hunk 2]
```diff
@@ -1,6 +1,128 @@
...
+    scanf("%d", &noOfDrinks);
+    getchar();
+    printf("Please input the drinks: each line in the format <drink> (price):\n");
+    drinks=(char**)malloc(noOfDrinks*sizeof(char*));
+    for(int i=0;i<noOfDrinks;i++)
+    {
+        drinks[i]=(char*)malloc(MAX_FOOD_NAME*sizeof(char));
+        readType(drinks[i]);
+        drinkPrices[i]=readPrice();
+
```
Comment (Ranapop) [inline main.c]: no need for space here

File: main.c [hunk 3]
```diff
@@ -1,6 +1,128 @@
...
+        printf("\n");
+    }
+    printf("The drinks data is:\n");
+    printf("drinks: ");
+    for(int i=0;i<noOfDrinks-1;i++)
+    {
+        printf("%s, ", drinks[i]);
+    }
+    printf("%s", drinks[noOfDrinks-1]);
+    printf("\n");
```
Comment (Ranapop) [inline main.c]: could be written in one printf call

File: main.c [hunk 4]
```diff
@@ -1,6 +1,128 @@
...
+        }
+        free(types[i]);
+        free(prices[i]);
+        free(foods[i]);
+    }
+    free(types);
+    free(prices);
+    free(foods);
+    free(noOfTypes);
+    for(int i=0; i<noOfTypes[i];i++)
```
Comment (Ranapop) [inline main.c:84]: why are you going to noOfTypes[i] here?

**Review by Ranapop — CHANGES_REQUESTED** [review]
see code comments

File: main.c (hunk 4 above)
Comment (moisaoana) [reply main.c:84]: If I go to noOfTypes, I get the warning 'comparison between pointer and integer'

**Pull request merged**